cmd/proxy/main.go

```diff
@@ -86,6 +87,14 @@ type ProxyRunOptions struct {
 	serverID string
 	// Number of proxy server instances, should be 1 unless it is a HA proxy server.
 	serverCount uint
+	// Agent pod's namespace for token-based agent authentication
+	agentNamespace string
+	// Agent pod's service account for token-based agent authentication
```

Comment:
Will we ever want different service accounts for different agents? Eg. agent service account per failure domain?